src/components/form/formControlsComponent.tsx

```diff
@@ -85,12 +85,13 @@ interface Option {
 }
 
 interface SelectProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+    dataTestId?: string
     hasError?: boolean
     options: Option[]
 }
 
-export const Select = styled(({ options, placeholder = '', ...props }: SelectProps) => (
-    <select {...props}>
```

Comment:
Fix React log warning in console due to invalid attribute `dataTestId` (should be `data-testid`) and `hasError` is not a valid attribute, it is used by `inputMixin` below.